examples/java/plugin-auto-refresh/README.MD

```diff
@@ -0,0 +1,222 @@
+
+## Get the example function
+
+Clone this repository which contains the example functions. 
+
+```bash
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance
+cd vcenter-event-broker-appliance/examples/java/plugin-auto-refresh
+git checkout master
+```
+
+## Customize the function
+
+Define the function-specific settings. They are performed in the `plugin-auto-refresh.yml` file. Open and edit the `plugin-auto-refresh.yml` provided within the Python plugin auto-refresh example code. Add the environmental variables `VCENTER_FQDN`, `VCENTER_THUMBPRINT`, `VCENTER_PORT` and `VCENTER_GUID`. Change `gateway` and (optionally) `topic` as per your environment/needs.
+
+```yaml
+provider:
+  name: openfaas
+  gateway: https://VEBA_FQDN_OR_IP/ # replace with your vCenter Event Broker Appliance environment
+functions:
+  plugin-auto-refresh:
+    image: vmware/plugin-auto-refresh:latest
+    environment:
+      write_debug: true
+      read_debug: true
+      VCENTER_FQDN: YOUR_VCENTER_FQDN
+      VCENTER_THUMBPRINT: YOUR_VCENTER_THUMBPRINT
+      VCENTER_PORT: YOUR_VCENTER_PORT
+      VCENTER_GUID: YOUR_VCENTER_GUID
+    annotations:
+      topic: HostConnectedEvent, HostDisconnectedEvent # the event can be changed depending on the plugin use cases
+```
+
+## Set up the function and the plugin
+
+### Deploy the function
+
+After you've performed the steps and modifications above, you can go ahead and deploy the function:
+
+```bash
+faas-cli deploy -f plugin-auto-refresh.yml --tls-no-verify
+Deployed. 202 Accepted.
+```
+
+### Port-forwarding
+Then you should forward the port that the plugin uses on your VEBA, so that you can connect to the plugin from outside. Log in to your VEBA machine and execute the following command:
+
+```bash
+kubectl port-forward --address 0.0.0.0 pod/<pod_name> 8089:8089 #The plugin runs on port 8089 by default. If you have changed the port, edit it in the command here.
+```
+
+### Allow http connections in your vSphere Client
+Ssh to your vCenter:
+
+```bash
+ssh root@YOUR_VC_IP
+```
+
+Then go to etc/vmware/vsphere-ui and add "allowHttp=true" to the end of the webclient.properties file.
+
+After that restart the client:
+
+```bash
+service-control --restart vsphere-ui
+```
+
+### Plugin registration
+Lastly, register the plugin to your vCenter followingin these steps:
+   1. Go to https://YOUR_VCENTER_IP/mob/?moid=ExtensionManager
+   2. Click `RegisterExtension`
+   3. Paste the extension:
+
+   ```xml
+   <extension>
+   <description>
+      <label>Remote Plug-in</label>
+      <summary>This is a sample plug-in</summary>
+   </description>
+   <key>com.vmware.sample.remote</key>
+   <company>VMware</company>
+   <version>1.0.0</version>
+   <server>
+      <url>http://PLUGIN_SERVER_IP:8089/sample-ui/plugin.json</url>
```

Comment:
There's no mentioning in the docs that describe what this placeholder refers to `PLUGIN_SERVER_IP`.
Can you please elaborate?

Reply:
Added in the README.md file.